powermax/models/masking_view.go

```diff
@@ -23,10 +23,10 @@ import "github.com/hashicorp/terraform-plugin-framework/types"
 type MaskingViewResourceModel struct {
 	Name           types.String `tfsdk:"name"`
 	ID             types.String `tfsdk:"id"`
-	StorageGroupId types.String `tfsdk:"storage_group_id"`
-	HostId         types.String `tfsdk:"host_id"`
-	HostGroupId    types.String `tfsdk:"host_group_id"`
-	PortGroupId    types.String `tfsdk:"port_group_id"`
+	StorageGroupID types.String `tfsdk:"storage_group_id"`
```

Comment:
Did you run the automated test against these? since they are populated by the `copy` method then the models needed to be the same as the model returned from OpenAPI. If you change these you might need to set them manually in the helper method.